src/ui/RunEditor.tsx

```diff
@@ -913,7 +919,7 @@ export class RunEditor extends React.Component<Props, State> {
 
                             expandedRow =
                                 <tr key={`${run.id}_expanded`} className={evenOdd}>
-                                    <td colSpan={4}>
+                                    <td colSpan={4 + (variables?.length || 0)}>
```

Comment:
I believe you can use ?? instead of || here, which is the new way to do it, as it doesn't apply to false-like values.